lms/static/certificates/sass/_print.scss

```diff
@@ -94,6 +94,12 @@
 // ------------------------------
 @media print {
 
+    html,
+    body {
+        height: 650px;
```

Comment:
Okay, turns out that using a fixed pixel value is not good. It might work on 8.5x11 but A4 still produces two sheets. Hmm.